CONTRIBUTOR-ROLE.md

```diff
@@ -0,0 +1,96 @@
+# Contributor Roles
+
+## Security Team
+
+Security Team members are listed in the Metal3 user-guide under the
+[Project Security Policy](https://book.metal3.io/security_policy#metal3-security-committee-members).
+The security team is responsible for responding to all security issues reported
+to Metal3 project. The team can be reached at `metal3-security@googlegroups.com`.
+(Please do not report any security vulnerability to the committee members
+directly.) The security team ensures that the issues are mitigated and
+a responsible disclosure process is followed in a reasonable time. The security
+team has a designated Security Lead. Currently,
+[Tuomo Tanskanen](https://github.com/tuminoid) is acting as the Metal3 Security
+Lead. Below are the responsibilities of the Security team.
+
+* Responsibilities:
+
+   * Thoroughly investigate all reports to the Security Team
+   * Does not share any vulnerability information with others unless necessary
+    to fix the issue
+   * Notify the reporter as the security issue moves through the identification
+    and resolution process
+   * Uphold the [Project Security Policy](https://book.metal3.io/security_policy#metal3-security-committee-members).
+
+* Qualifications:
+
+   * Has a solid grasp of the overall architecture of Metal3, its usage in
+    production environments, and its threat model.
+   * Established the trust of Metal3 community members that they will uphold the
+    security posture of Metal3
+
+* Privileges:
+
+   * Access to security reports
+   * Membership of the security team mailing list
+   * Security Lead has elevated access to Metal3 github organization through
+   the `metal3_security_team` GitHub team.
+
+The process of becoming a security team member is:
+
+1. Nominated by a security team member and seconded by a maintainer
```

Comment:
```suggestion
1. Nominated by the Security Lead and seconded by an Metal3 Org Admin
```
I think any member along with any maintainer is not right.

Reply:
in that case we need to add another role Metal3 Org Admin. I wanted to avoid fine grained classification for now.

Reply:
Good point. We have this role right now, so we need to document it as well.

Reply:
Would it be ok that I add an issue here and we discuss the criteria to be selected for that role within maintainers and add it later?

Reply:
That role is not very relevant for a regular contributor, we can add it later. Will you incorporate the original comment anyways?